### ISSUE TITLE
WCAG (Web Content Accessibility Guidelines) violations reported as Critical and Serious, while using Select2 4.0.13

### ISSUE THREAD

**Issue by srinadh** [select2/select2#6205]
WCAG (Web Content Accessibility Guidelines) violations reported as Critical and Serious, while using Select2 4.0.13

**Description of Problem**:
When used Select2 4.0.13 along with jquery-ui.js (v1.13.1), there are < ul > and < li > elements relevant WCAG violations are popping up when verifying using AXE Devtool (or) similar tool. In a summary, < ul > element added with role="tablist", however, < li > element that was child to < ul > element not added by select2 with role="tab".

2 Major violations occurring are - 
1. Certain ARIA roles must contain particular children
2. < li > elements must be contained in a < ul > or < ol >

Problematic code snippet for us in select2.full.js is -
```
Search.prototype.render = function (decorated) {
    var $search = $(
      '<li class="select2-search select2-search--inline">' +
        '<input class="select2-search__field" type="search" tabindex="-1"' +
        ' autocomplete="off" autocorrect="off" autocapitalize="none"' +
        ' spellcheck="false" role="searchbox" aria-autocomplete="list" />' +
      '</li>'
    );

    this.$searchContainer = $search;
    this.$search = $search.find('input');

    var $rendered = decorated.call(this);

    this._transferTabIndex();

    return $rendered;
  };
```
Can we get suggestion on possibilities to overcome these violations in perspective of select2 (or) does these violations fixed in near time in upcoming select2 4.1.0 release !

**Comment by kevin-brown** [comment]
Many accessibility issues are handled in 4.1.0. I would strongly advise you to upgrade to the latest release candidate.

**Comment by srinadh** [comment]
When we can expect select2 4.1.0 release candidate (Pre-release) move into final/production release. i.e. Do we know approximate select2 4.1.0 final/production release time!
